daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
these are not vars. But it is a return value. being used in `registerSubRouter` to register the http handler for the `/networks` path.